fe1-web/src/features/wallet/objects/__tests__/Wallet.test.ts

```diff
@@ -0,0 +1,51 @@
+import { expect } from 'chai';
```

Comment:
This is not the `expect` we use. As I said in my first comment, you should use jest.

Reply:
Right, I forgot about this one :(